api/applications.go

```diff
@@ -283,3 +291,69 @@ func ApplicationStatus(accountID int, applicationID int, moduleName string) (as
ror.log to STDERR."},{"timestamp":"2020-12-22T11:24:45.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"Listening on http://0.0.0.0:9000/metrics"},{"timestamp":"2020-12-22T11:24:45.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"[WARN] Updating environment"},{"timestamp":"2020-12-22T11:24:47.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"> learn-starter@0.1.0 start /opt/section/node"},{"timestamp":"2020-12-22T11:24:47.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"> next start /opt/section/node -p 8080"},{"timestamp":"2020-12-22T11:24:47.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"info  - Loaded env from /opt/section/node/.env.production"},{"timestamp":"2020-12-22T11:24:47.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"ready - started server on http://localhost:8080"},{"timestamp":"2020-12-22T11:24:49.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"[WARN] The node app is already being served."},{"timestamp":"2020-12-22T11:24:49.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"[WARN] Initialising Nginx"},{"timestamp":"2020-12-22T11:24:49.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"[WARN] Updating custom Nginx server.conf"},{"timestamp":"2020-12-22T11:24:15.335Z","source":"testjs-d97hd-4k4", "type":"app","message":" ERROR  Error: Request failed with status code 403"},{"timestamp":"2020-12-22T11:24:15.335Z","source":"testjs-d97hd-4k4", "type":"app", "message":"    at settle (/opt/section/node/node_modules/axios/lib/core/settle.js:17:12)"},{"timestamp":"2020-12-22T11:24:15.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"    at endReadableNT (_stream_readable.js:1092:12)"},{"timestamp":"2020-12-22T11:24:15.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"    at IncomingMessage.emit (events.js:187:15)"},{"timestamp":"2020-12-22T11:24:15.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"    at createError (/opt/section/node/node_modules/axios/lib/core/createError.js:16:15)"},{"timestamp":"2020-12-22T11:24:15.335Z","source":"testjs-d97hd-4k4", "type":"app","message":"    at IncomingMessage.handleStreamEnd (/opt/section/node/node_modules/axios/lib/adapters/http.js:236:11)"},{"timestamp":"2020-12-22T11:24:49.335Z","source":"testjs-d97hd-4k4", "type":"access","message":"method=GET path=/_next/static/chunks/main.js time_taken_ms=18 status=200 bytes_sent=7405 host=www.varnishdemo.com section_io_id=e47a44a74364e984eb4b75c3c0ec908"},{"timestamp":"2020-12-22T11:24:51.335Z","source":"testjs-d97hd-4k4", "type":"access","message":"method=GET path=/ time_taken_ms=1800 status=200 bytes_sent=1024 host=www.varnishdemo.com section_io_id=52ad30f0f33a6ed11b5693c63e5b3ea4"},{"timestamp":"2020-12-22T11:24:52.335Z","source":"testjs-d97hd-4k4", "type":"access","message":"method=GET path=/_next/static/chunks/framework.js time_taken_ms=11 status=200 bytes_sent=3405 host=www.varnishdemo.com section_io_id=981d39903a281a5f896517a364b82a80"}]}}`)
```

Comment:
You can put this JSON payload in a file under a `testdata` folder to use with unit tests.